build-tools/scripts/get-git-branch.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash -e
+SUBMODULE_NAME="$1"
```

Comment:
This file should contain a comment mentioning that it should be kept in sync with `GitCommitInfo.cs`.